legacy/jacorb/src/main/java/org/jboss/as/jacorb/logging/JacORBLogger.java

```diff
@@ -445,4 +445,13 @@
                 "and 'org.omg.PortableInterceptor.ORBInitializerClass.org.jboss.as.jacorb.csiv2.SASClientInitializer' properties to be \"\"")
     String cannotUseSecurityClient();
 
+    @Message(id = 132, value = "Property %s cannot be emulated using OpenJDK ORB and is not supported")
+    OperationFailedException cannotEmulateProperty(String property);
+
+    @LogMessage(level = WARN)
+    @Message(id = 133, value = "Wildfly does not use JacORB as an ORB implementation anymore. JacORB subsystem would be "
```

Comment:
Please don't use "WildFly" in messages, as it won't be WildFly once it's EAP. Use the passive voice to avoid the naming issue, i.e. "JacORB is not used as an ORB implementation any more."